src/lib/features/feature-lifecycle/feature-lifecycle-service.ts

```diff
@@ -134,7 +134,7 @@ export class FeatureLifecycleService extends EventEmitter {
             }
             if (env.type === 'production') {
                 await this.stageReceivedMetrics(features, 'live');
-            } else if (env.type === 'development') {
+            } else if (env.type !== 'production') {
```

Comment:
This is weird, this is just else clause.

Also I think pre-live is also production, but toggle turned off.

So I would do

```
await this.stageReceivedMetrics(features, 'pre-live');
if (env.type === 'production') {
    await this.stageReceivedMetrics(features, 'live');
} 
```

Reply:
This means, any metrics will put it to pre-live, any metrics in production puts it in production.

Reply:
nice idea